src/data/multiset/basic.lean

```diff
@@ -2091,6 +2091,9 @@ begin
   rw [← count_add, sub_add_inter, count_sub, tsub_add_min],
 end
 
+lemma count_sup (a : α) (s t : multiset α) :
+  count a (s ⊔ t) = max (count a s) (count a t) := by simp
+
```

Comment:
This lemma is a duplicate of `multiset.count_union`, there is no need to have both. If you have a statement about `⊔` on multisets, you should use `dsimp` to convert it into `\union` before searching for a lemma.